suite-native/message-system/src/components/MessageSystemBannerRenderer.tsx

```diff
@@ -1,31 +1,12 @@
-import { useSafeAreaInsets } from 'react-native-safe-area-context';
-import { useSelector } from 'react-redux';
-
 import { VStack } from '@suite-native/atoms';
-import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { selectActiveBannerMessages } from '@suite-common/message-system';
 
 import { MessageBanner } from './MessageBanner';
 
-const messageBannerContainerStyle = prepareNativeStyle<{ topSafeAreaInset: number }>(
```

Comment:
I don't think it is a good idea to remove the top area inset completely. Because IMO now if there would be any banners rendered, they would render underneath the StatusBar.

Reply:
What about rendering rendering the whole container conditionally?

Reply:
Yes, I will do that. I'll prepare fixup tomorrow morning ideally.